io.edgehog.devicemanager.OSInfo.json

```diff
@@ -0,0 +1,19 @@
+{
+  "interface_name": "io.edgehog.devicemanager.OSInfo",
```

Comment:
can you provide any example here in the comments, about what `/osName` and `/osVersion` would look like on a raspberry running Raspbian, and on a ESP32 board?